project/src/components/places-list/places-list.tsx

```diff
@@ -1,13 +1,20 @@
+import { Offer } from '../../types/offer';
 import PlaceCard from '../place-card/place-card';
+import {useState} from 'react';
 
-type Cards = {
-  list: number[]
-};
+type Offers = {
+  list: Offer[]
+}
+
+function PlacesList({list}: Offers): JSX.Element {
+  const [activeCard, setActiveCard] = useState({});
```

Comment:
Значение не нужно, опусти его при деструктуризации
```suggestion
  const [, setActiveCard] = useState({});
```